docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md

```diff
@@ -1 +1,87 @@
-# DR Solution Based on Cross-Region Deployment in a Single Cluster
\ No newline at end of file
+# DR Solution Based on Cross-Region Deployment in a Single Cluster
+
+## How Disaster Recovery Works in GreptimeDB
+GreptimeDB is well-suited for cross-region disaster recovery. Customers may have varying regional characteristics and business needs, and GreptimeDB offers tailored solutions to meet these diverse requirements.
+
+GreptimeDB resource management involves the concept of AZs (Availability Zones). An AZ is a logical unit of disaster recovery.
+It can be a DC(Datra Center), a compartment of a DC. This depends on the customer's specific DC conditions and deployment design.
+
+A GreptimeDB region is a city. When two DC are in the same region and one DC becomes unavailable, the other DC can take over the services of the unavailable DC. This is a localization strategy.
+
+### Disaster Recovery Solution: Across 2 regions, Data in the same region
+
+In this solution, the data is in one region (2 DCs), while the metadata across 2 regions.
+
+DC1 and DC2 are used together to handle read and write services, while DC3 (located in region2) is a replica used to meet the majority protocol. This architecture is also called the "2-2-1" solution.
+
+Both DC1 and DC2 must be able to handle all requests in extreme situations, so please ensure that sufficient resources are allocated.
+
+Features:
+    - 2ms latency in the same region
+    - 30ms latency in two regions
+
+Supports High Availability:
+    - A single AZ is unavailable with the same performance
+    - A single DC is unavailable with almost the same performance
+
+
+![DR-across-2dc-1region](/DR-across-2dc-1region.png)
```

Comment:
It is recommended to move this line to line 12. Seeing an image first will make it easier for readers to understand the content.